elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -158,7 +160,11 @@ private static void from15_1(ChainedTransformationDescriptionBuilder chainedBuil
                 .setDiscard(DiscardAttributeChecker.UNDEFINED, CREDENTIAL_STORE)
                 .setDiscard(DiscardAttributeChecker.UNDEFINED, SECRET_KEY)
                 .addRejectCheck(RejectAttributeChecker.DEFINED, CREDENTIAL_STORE)
-                .addRejectCheck(RejectAttributeChecker.DEFINED, SECRET_KEY);
+                .addRejectCheck(RejectAttributeChecker.DEFINED, SECRET_KEY)
```

Comment:
We'll need another Elytron subsystem bump. Will let you know once I've submitted a PR for that.

Reply:
@Ashpan Just FYI, the Elytron subsystem bump is now available here:

https://github.com/wildfly/wildfly-core/pull/5090